darwin/utils.py

```diff
@@ -168,25 +169,62 @@ def persist_client_configuration(
     return config
 
 
-def parse_darwin_json(path: Path, count: int):
+def get_local_filename(metadata: dict):
+    filename = Path(metadata["filename"])
+    original_filename = Path(metadata["original_filename"])
+    return f"{filename.stem}_{original_filename.stem}{filename.suffix}"
+
+
+def parse_darwin_json(path: Union[str, Path], count: int):
+    path = Path(path)
     with path.open() as f:
         data = json.load(f)
         if not data["annotations"]:
             return None
-        annotations = list(filter(None, map(parse_darwin_annotation, data["annotations"])))
-        annotation_classes = set([annotation.annotation_class for annotation in annotations])
-
-        return dt.AnnotationFile(
-            path,
-            data["image"]["original_filename"],
-            annotation_classes,
-            annotations,
-            data["image"]["width"],
-            data["image"]["height"],
-            data["image"]["url"],
-            data["image"].get("workview_url"),
-            data["image"].get("seq", count),
-        )
+        if "fps" in data["image"] or "frame_count" in data["image"]:
```

Comment:
This makes perfect sense with the way we currently export videos.
As we discussed separately, it's probably worse including the item type within the format, so we don't have to do this.